packages/eslint-plugin/docs/rules/require-await.mdx

```diff
@@ -12,6 +12,10 @@ import TabItem from '@theme/TabItem';
 This rule extends the base [`eslint/require-await`](https://eslint.org/docs/rules/require-await) rule.
 It uses type information to allow promise-returning functions to be marked as `async` without containing an `await` expression.
 
+:::note
+Unlike the base rule, our implementation also supports [async generator functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function*).
```

Comment:
It's not 100% unambiguous what it means to "support" generator functions (and it's tricky to describe in words). Would you mind adding one example to the docs for the generator function case? 🙂

Reply:
You want to delete the note and add an example instead?

Reply:
Nah, definitely need the words too 👍

Reply:
I added a new example.